test/integration/auth/mongodb_oidc_k8s.prose.07.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
+
+import { type Collection, MongoClient } from '../../mongodb';
+
+const DEFAULT_URI = 'mongodb://127.0.0.1:27017';
+
+describe('OIDC Auth Spec K8s Tests', function () {
+  // Note there is no spec or tests for GCP yet, these are 2 scenarios based on the
+  // drivers tools scripts available.
```

Comment:
Is this still accurate?  

Also - this isn't always testing GCP is it?  We run in on aks, eks and GCP.  Maybe we can update the describe below so that its accurate?

Reply:
I've updated the comment now to reflect accurately what the test runs on.